tests/app/main/test_briefs.py

```diff
@@ -2297,3 +2332,20 @@ def test_supplier_details_shown(self, data_api_client):
         assert requirements_data.row(1).cell(1) == "02/02/2017"
         assert requirements_data.row(2).cell(0) == "Email address"
         assert requirements_data.row(2).cell(1) == "contact@big.com"
+
+    def test_visit_to_start_page_redirects_to_result_page(self, data_api_client):
```

Comment:
I think you're right about the flash message, it makes a nice distinction about the thing that's actually being tested which is more robust than the other approach.

I redid that last commit so it works like this now which should also cover the case of a visit to a question page for a completed response.